server/controllers/customers/csv-customers.js

```diff
@@ -0,0 +1,28 @@
+const Customer = require('../../models/Customer')
+const handleCSV = require('../../utilities/handle-csv')
+
+// Upload and save CSV file to database
+const upload = async (req, res) => {
+  try {
+    if (req.file == undefined) return res.status(400).send("Please upload a CSV file!")
```

Comment:
nit: this if only contains an if condition, and you can change this to be an inline condition by doing this.
```
req.file && res.status(400).send("Please upload a CSV file!")
```
Otherwise, if you prefer the if condition, it is generally better to use to leave the curly brackets. It makes it easier to see what is the context of the if block.